prepare-module.sh

```diff
@@ -10,14 +10,18 @@ set -u
 set -o pipefail
 
 mkdir -p lambda-files
+echo "* Compiling TypeScript sources"
 npm run-script build
 cp package.json lambda-files
 cd lambda-files || exit "Directory 'lambda-files' does not exist"
+echo "* Installing Javascript dependencies for Lambda"
 npm install --no-package-lock --production
+echo "* Removing test folder and package.json from Lambda source files"
+rm -rf test
```

Comment:
This removes `test` folder from dist folder, I assume. How did the `test` folder end up into the dist folder in the first place?

Reply:
The test folder ends up there currently because we are currently using the same tsconfig.json file both for the actual Lambda code and its test code. So when you run `tsc` you end up with the compiled test files in the outDir as well. I think we could avoid this by having a separate tsconfig.json for the tests but didn't feel like spending time setting it up when I could just simply remove the compiled tests file in the `prepare-module.sh` script.